pkg/utils/protoutils/marshal.go

```diff
@@ -143,6 +140,18 @@ func UnmarshalMapToProto(m map[string]interface{}, into proto.Message) error {
 	return jsonpb.Unmarshal(bytes.NewBuffer(data), into)
 }
 
+func UnmarshalStruct(structuredData *structpb.Struct, into interface{}) error {
```

Comment:
function looks fine - why do we need it here? i dont see it used in the PR?

Reply:
It's used in Gloo. Essentially the go-utils protoutils will be unusable by gloo after this since they are all gogo proto. But other projects still using gogo will need those protoutils to stay the same. So I'm just updating these to have the go proto versions

Reply:
Going to remove though, because it also exists in gloo already